apps/me/src/app/app-routing.module.ts

```diff
@@ -0,0 +1,30 @@
+import { NgModule } from '@angular/core';
+import { PreloadAllModules, RouterModule, Routes } from '@angular/router';
+import { environment } from '@env/environment';
+
+const routes: Routes = [
+  {
+    path: 'main',
+    loadChildren: () =>
+      import(`./main/main.module`).then(module => module.MainModule),
+  },
+  {
+    path: '',
+    redirectTo: '/main',
```

Comment:
```suggestion
    redirectTo: '/home',
```
Adapt to new path name from above.